### PR TITLE
Remove formatting preferences from .ensime

### PR DESCRIPTION
**Pull request by yyadavalli**
Plugin counterpart for https://github.com/ensime/ensime-server/pull/1619

### CODE REVIEW

File: src/sbt-test/sbt-ensime/scalariform-only/project/plugins.sbt [hunk 1]
```diff
@@ -1 +0,0 @@
-../../scalariform/project/plugins.sbt
```
Comment (fommil) [inline src/sbt-test/sbt-ensime/scalariform-only/project/plugins.sbt:1]: this looks like a mistake, it should just use the symbolic link

**Review by fommil — CHANGES_REQUESTED** [review]

File: src/sbt-test/sbt-ensime/scalariform-only/project/plugins.sbt (hunk 1 above)
Comment (yyadavalli) [reply src/sbt-test/sbt-ensime/scalariform-only/project/plugins.sbt:1]: Previous, the file was a symbolic link to the `plugins.sbt` in `scalariform` test but that test was no longer needed and I deleted it, so moved the file here

Comment (fommil) [reply src/sbt-test/sbt-ensime/scalariform-only/project/plugins.sbt:1]: oh I see, I'm silly :blush:

**Comment by fommil** [comment]
releasing v1.11.1

**Comment by yyadavalli** [comment]
You can wait for watch disabling to do that

**Comment by fommil** [comment]
oh, too late :smile: we can do the next tomorrow

**Pull request merged**